__tests__/app.test.js

```diff
@@ -353,4 +353,40 @@ test("404: Responds with error message if article does not exist", () => {
     expect(body.msg).toBe("Article Doesn't Exist")
   })
 })
+})
+describe("DELETE /api/comments/:comments_id", () => {
+  test("204: Should delete the comment with the given ID", () => {
+    return request(app)
+    .delete("/api/comments/1")
+    .expect(204)
+    .then(() => {
+      return db.query(`SELECT * FROM comments WHERE comment_id = $1`, [1]).then(({ rows }) => {
+        expect(rows.length).toBe(0)
+      })
+    })
+  })
+  test("200: Should return a message when the comment has been successfully deleted", () => {
+    return request(app)
+    .delete("/api/comments/2?return_message=true")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Comment successfully deleted")
+    })
+  })
```

Comment:
This is an interesting behaviour, but it goes against convention as a delete endpoint Should give back a 204, so this giving back a 200 is not expected behaviour even with an added query